CITATION.cff

```diff
@@ -0,0 +1,43 @@
+cff-version: 1.2.0
+message: "Cite this paper whenever you use Julia"
+authors:
+- family-names: "Bezanson"
+  given-names: "Jeff"
+- family-names: "Edelman"
+  given-names: "Alan"
+- family-names: "Karpinski"
+  given-names: "Stefan"
+- family-names: "Shah"
+  name-particle: "B"
+  given-names: "Viral"
+title: "Julia: A fresh approach to numerical computing"
+version: "v1"
+license: "MIT"
+doi: "10.1137/141000671"
+date-released: 2017-02-07
+url: "https://github.com/JuliaLang/julia"
+references:
+  - scope: "Cite this paper whenever you use Julia"
+    type: article
+    title: "Julia: A fresh approach to numerical computing"
+    authors:
+      - family-names: "Bezanson"
+        given-names: "Jeff"
+      - family-names: "Edelman"
+        given-names: "Alan"
+      - family-names: "Karpinski"
+        given-names: "Stefan"
+      - family-names: "Shah"
+        name-particle: "B"
+        given-names: "Viral"
+    doi: "10.1137/141000671"
+    journal: "SIAM Review"
+    volume: 59
+    number: 1
+    pages: 33
+    start: 65
+    end: 98
+    year: 2017
+    url: "https://epubs.siam.org/doi/10.1137/141000671"
```

Comment:
Should this `url` be `https://epubs.siam.org/doi/10.1137/141000671` or `https://dx.doi.org/10.1137/141000671`?

The latter redirects to the former, and in theory the latter is more "future-proof".

Reply:
Yes, preferable to use the doi.org URL.

Reply:
```suggestion
    url: "https://dx.doi.org/10.1137/141000671"
```